support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx

```diff
@@ -255,6 +259,27 @@ function WeeklyCheckoutForm(props: PropTypes) {
 		  }
 		: undefined;
 
+
+	const country = countryGroups[props.countryGroupId].supportInternationalisationId;
+	
+	const COOKIE_EXPIRY_DAYS = 3;
+	
+	useAbandonedBasketCookie(
+		props.product,
+		props.price.price,
+		tierBillingPeriodName,
+		country,
+		COOKIE_EXPIRY_DAYS
+	);
+		
+	useEffect(() => {
```

Comment:
test code?

Reply:
sorry?

Reply:
you had some manual testing code leftover at this point in time, console logging etc

Reply:
Ah thanks, this is all sorted now